po/de.po

```diff
@@ -16,32 +16,31 @@ msgstr ""
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 "Plural-Forms: nplurals=2; plural=n != 1;\n"
-"X-Generator: Poedit 3.4.2\n"
+"X-Generator: Poedit 3.5\n"
 
 #.
 #: src/widgets/sidebar.vala:32
 msgid "Only display the runners that are currently installed on your system"
 msgstr ""
+"Nur Ausführungswerkzeuge anzeigen die aktuell auf dem System installiert sind"
 
 #.
 #: src/widgets/sidebar.vala:35
-#, fuzzy
 msgid "Only Installed"
 msgstr "Nur installierte"
```

Comment:
`Nur Installierte` (capitalization). It's meant to be in Title Case for the UI button. If that makes sense in German, we should do it.

Reply:
done